amz-review-scraper/test_url.py

```diff
@@ -0,0 +1,6 @@
+import pytest
```

Comment:
so you want to have a folder called test, that sits parallel to the amz-review-scaper folder.
then you want to copy the file you wants to tests name, and add test to the beginning.

so `urls` would be `test_urls`. Then you can run all the tests for your project with `pytest test`